Cython/Compiler/Options.py

```diff
@@ -189,6 +189,7 @@ def copy_inherited_directives(outer_directives, **new_directives):
     'cdivision_warnings': False,
     'c_api_binop_methods': False,  # was True before 3.0
     'overflowcheck': False,
+    'noexcept': False,
```

Comment:
Options.py also has a place where you set where directives can be applied (`directive_scopes`). I think `noexcept` should be `module` only.

Reply:
Thanks. Fixed that.